xml/metering-metering_metertypes.xml

```diff
@@ -116,15 +116,9 @@
   <title>&productname; Default Meters</title>
   <para>
    These meters are installed and enabled by default during an &productname;
-   installation.
-  </para>
-  <para>
-   Detailed information on the Ceilometer API can be found on the following
-   page:
-  </para>
-  <para>
-   <link xlink:href="http://docs.openstack.org/developer/ceilometer/webapi/v2.html">Ceilometer
-   Web API</link>.
+   installation. More information about Ceilometer can be found at <link
+   xlink:href="http://docs.openstack.org/developer/ceilometer/webapi/v2.html">OpenStack
+   Ceilometer</link>.
```

Comment:
This redirects to https://docs.openstack.org/ceilometer/latest/  Would that be a better url to use?

Reply:
Thanks for checking that...better. fixed and force-pushed.